payments-core/src/main/java/com/stripe/android/FraudDetectionDataRepository.kt

```diff
@@ -32,28 +35,26 @@ internal interface FraudDetectionDataRepository {
     fun save(fraudDetectionData: FraudDetectionData)
 }
 
+private val timestampSupplier: () -> Long = {
+    Calendar.getInstance().timeInMillis
```

Comment:
How does this compare to using `System.currentTimeMillis()` or `Date().getTime()` ?

Reply:
Some SO search led me to [this](https://stackoverflow.com/a/368130/802372) answer, tldr seems Calendar is the slowest one. Would rather keep it for now as this line is copied from the deleted `[DefaultFradDetectionDataRequestExectuor](https://github.com/stripe/stripe-android/blob/8d04912297dc8580b347160141698a2a92b10c0a/payments-core/src/main/java/com/stripe/android/networking/FraudDetectionDataRequestExecutor.kt#L22)`